flutter/tonic/dart_state.h

```diff
@@ -52,6 +53,11 @@ class DartState {
   DartLibraryLoader& library_loader() { return *library_loader_; }
   DartMessageHandler& message_handler() { return *message_handler_; }
 
+  void set_isolate_reloader(DartIsolateReloader* isolate_reloader) {
+    isolate_reloader_ = std::unique_ptr<DartIsolateReloader>(isolate_reloader);
```

Comment:
The `std::unique_ptr` should be created by the code that allocates the object.  This function should take a `std::unique_ptr<DartIsolateReloader>` as an argument and `std::move` the argument into `isolate_reloader_`